package.json

```diff
@@ -32,13 +32,13 @@
     "typescript": "5.1.6"
   },
   "scripts": {
-    "_compile": "pnpm --filter merkle-tree-wasm compile && pnpm --filter spartan-ecdsa-wasm compile",
```

Comment:
does not work if wasm-pack is run in parallel for all the packages. (Don't know why)